ui/index.html

```diff
@@ -0,0 +1,13 @@
+<!doctype html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <link rel="icon" type="image/x-icon" href="favicon.ico">
```

Comment:
Seems like we should use something other than .ico. Maybe svg? This doesn't work best on mobile devices, but would be good enough.